clustered/integration-test/src/test/java/com/tc/logging/TestLoggingService.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.tc.logging;
+
+import java.net.URI;
+
+/**
+ * A Terracotta logger writing to {@code System.out}.
+ */
+public class TestLoggingService implements TCLoggingService {
```

Comment:
This was added to get client logging from `integration-test` module tests.  Without this, the client logging winds up in the bit bucket _or_ requires a JDK logging configuration.